.github/.wordlist.txt

```diff
@@ -1327,6 +1327,7 @@ ureverse
 URI
 urlrequest
 usbmodem
+usbmodemXXXXXXX
```

Comment:
You should be able to avoid these things now if the md clearly marks these as code blocks using backticks.

Reply:
ok, thanks!

Reply:
Please remove the non-words from wordlist and use only backticks for code (and that should make spellcheck happy)